harness/determined/exec/prep_container.py

```diff
@@ -61,6 +61,11 @@ def trial_prep(info: det.ClusterInfo, cert: certs.Cert) -> None:
         model_def.extractall(path=constants.MANAGED_TRAINING_MODEL_COPY)
         model_def.extractall(path=".")
 
+    # pre-0.18.2 code wrote tensorboard stuff under /tmp/tensorboard
+    tensorboard_path_on_chief = f"/tmp/tensorboard-{info.allocation_id}-0"
+    os.makedirs(tensorboard_path_on_chief, exist_ok=True)
+    os.symlink(src=tensorboard_path_on_chief, dst="/tmp/tensorboard")
```

Comment:
This will raise a os.FileExists error any time that the user has `/tmp` mounted (either manually or with slurm).

I would recommend in the os.FileExists case, we override what's there, because I think that will result in the least offensive behavior; it would be annoying if somebody bind-mounted /tmp and the same symlink persisted through many, many trials.